src/components/MovieCard/MovieCard.js

```diff
@@ -1,14 +1,22 @@
-import './MovieCard.css'
+import "./MovieCard.css";
+import PropTypes from "prop-types";
 
-const MovieCard = ({handleClick, id, poster_path, title, release_date}) => {
+const MovieCard = ({ handleClick, id, poster_path, title, release_date }) => {
   return (
-    <div className='card' onClick={() => handleClick(id)} id={id}>
-      <img className='image' src={poster_path}/>
+    <div className="card" onClick={() => handleClick(id)} id={id}>
+      <img className="image" src={poster_path} />
```

Comment:
Be sure to include an `alt` tag with specific information any time you're rendering an `img`